django_project/feti/api_views/feedback.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from feti.models.feedback import Feedback
+
+from feti.serializers.feedback_serializer import FeedbackSerializer
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '22/10/17'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+class FeedbackApiView(APIView):
```

Comment:
I am not sure if we need this. We can see all feedbacks in the admin view though.